.github/workflows/changelog.yml

```diff
@@ -23,7 +23,6 @@ concurrency:
 jobs:
   changelog:
     runs-on: ubuntu-latest
-    if: ${{ !contains(github.event.pull_request.labels.*.name, 'dependencies') && !contains(github.event.pull_request.labels.*.name, 'Skip Changelog') && !contains(github.event.pull_request.title, '[chore]')}}
```

Comment:
I'd suggest leaving something like the following to avoid runing changelog checks on dependabot PRs

```
if: ${{ github.actor != 'dependabot[bot]' }}
```